drivers/hmis/app/models/hmis/hud/event.rb

```diff
@@ -11,4 +11,39 @@ class Hmis::Hud::Event < Hmis::Hud::Base
   self.sequence_name = "public.\"#{table_name}_id_seq\""
 
   belongs_to :enrollment, **hmis_relation(:EnrollmentID, 'Enrollment')
+
+  def self.events_enum_map
+    Hmis::FieldMap.new(
+      ::HUD.events.map do |value, desc|
+        {
+          key: desc,
+          value: value,
+          desc: desc,
+        }
+      end,
+      include_base_null: false,
+    )
+  end
+
+  def self.referral_result_enum_map
+    Hmis::FieldMap.new(
+      [1, 2, 3].map do |value|
+        desc = ::HUD.referral_result(value)
+        {
+          key: desc,
+          value: value,
+          desc: desc,
+        }
+      end,
+      include_base_null: false,
+    )
+  end
+
+  def self.prob_sol_div_rr_result_enum_map
+    Hmis::FieldMap.yes_no_missing
+  end
+
+  def self.referral_case_manage_after_enum_map
+    Hmis::FieldMap.yes_no_missing
+  end
```

Comment:
This is what I'm seeing in the schema dump

```
+enum ReferralCaseManageAfter {
+  """
+  Data not collected*
+  """
+  DATA_NOT_COLLECTED_
+
+  """
+  No
+  """
+  NO
+
+  """
+  Yes
+  """
+  YES
+}
```

Reply:
Same for ProbSolDivRRResult

Reply:
@gigxz That's because the string it's based on is `"Data not collected*"`, which comes directly from `::HUD`. I've erred on the side of not changing descriptions, so I've left the trailing underscore in place

Reply:
Oh weird. Looks like a typo anyway, it should be fine to change. @eanders OK to change this line?
https://github.com/greenriver/hmis-warehouse/blob/8202a8644999d48ace2c10c448816fc59ffb675e/lib/util/hud.rb#L338

Reply:
I think that's ok.  We may get failures when we try to submit some random report, but I think we specify the row/column titles explicitly in the reports.
Go for it.

Reply:
@eanders So I should go ahead and remove the asterisk on that line?

Reply:
yeah, any of those that end in asterisk can be cleaned up.  Thanks